dev-client/src/model/elevation/elevationService.ts

```diff
@@ -18,6 +18,7 @@
 import {formatCoordinate} from 'terraso-mobile-client/util';
 
 const OUT_OF_RANGE_MESSAGE = 'Invalid or missing input parameters.';
+const UNABLE_TO_PROCESS_MESSAGE = 'Unable to complete operation.';
```

Comment:
Question: Are these messages that can be returned by the API? If so, curious if they are documented somewhere? I didn't see them mentioned on https://epqs.nationalmap.gov/v1/docs

Reply:
Yes, they are. I'm sure there are more; these are the ones we've found so far.